tests/test_unittest/test_gtc/test_gtir.py

```diff
@@ -137,66 +125,41 @@ def test_no_horizontal_offset_allowed(assign_stmt_with_offset):
 
 def test_symbolref_without_decl():
     with pytest.raises(ValidationError, match=r"Symbols.*not found"):
-        StencilBuilder().add_par_assign_stmt(
-            ParAssignStmtBuilder("out_field", "in_field").build()
-        ).build()
+        StencilFactory(params=[])
```

Comment:
nitpick: Here the crucial feature under testing of the stencil is hidden in the factory. I would prefer to make the assignment without symbol declaration explicit, even if it is redundant now (factory might also change down the line).

Reply:
Good catch! Will fix…